pkg/mic/mictest/mic_test.go

```diff
@@ -80,48 +80,51 @@ func TestSimpleMICClient(t *testing.T) {
 	eventCh <- aadpodid.PodCreated
 	go micClient.Sync(exit)
 	time.Sleep(2 * time.Second)
-	testPass := false
+	testPass := true
 	listAssignedIDs, err := crdClient.ListAssignedIDs()
 	if err != nil {
 		glog.Error(err)
-		panic("list assigned failed")
+		t.Errorf("list assigned failed")
 	}
 	if listAssignedIDs != nil {
 		for _, assignedID := range *listAssignedIDs {
 			if assignedID.Spec.Pod == "test-pod" && assignedID.Spec.PodNamespace == "default" && assignedID.Spec.NodeName == "test-node" &&
 				assignedID.Spec.AzureBindingRef.Name == "testbinding" && assignedID.Spec.AzureIdentityRef.Name == "test-id" {
-				testPass = evtRecorder.Validate(&LastEvent{Type: "Normal", Reason: "binding applied",
-					Message: "Binding testbinding applied on node test-node for pod test-pod-default-test-id"})
-				if !testPass {
-					panic("event mismatch")
-				}
+				/*
```

Comment:
need this ?

Reply:
I am slowly changing the way we test. This part is now un needed because it was an abstraction exposed by mic. I will remove the commented code as finalize the PR